src/ngraph/pass/memory_layout.cpp

```diff
@@ -38,16 +38,48 @@ bool pass::MemoryLayout::run_on_function(shared_ptr<ngraph::Function> function)
     MemoryManager mm(m_alignment);
     for (shared_ptr<Node> node : function->get_ordered_ops())
     {
+        std::map<descriptor::Tensor*, descriptor::Tensor*> inplace_outputs;
+        std::set<descriptor::Tensor*> reused_inputs;
+
+        if (auto op = std::dynamic_pointer_cast<op::Op>(node))
+        {
+            auto op_annotations = std::dynamic_pointer_cast<op::Op>(node)->get_op_annotations();
+            if (op_annotations)
```

Comment:
You previously put the variable assignment in the conditional statement on line 44. Can you do that here too? Consistency.